data/settings.js

```diff
@@ -1469,6 +1558,17 @@ plugin {
 \`\`\``
 	},
 
+	imapsieve_expunge_discarded: {
+		plugin: 'imap-sieve',
+		seealso: [ '[[plugin,sieve-imapsieve]]' ],
+		values: setting_types.BOOLEAN,
+		default: 'no',
+		text: `
+This setting determines whether de IMAPSieve plugin implicitly also expunges
+messages that were discarded by the executed Sieve script sequence; i.e., Sieve
+yielded no (implicit) keep.`
+	},
+
 	/* sieve_extprograms plugin */
 
 	'sieve_<extension>_socket_dir': {
```

Comment:
This should just be expanded/duplicated to `sieve_pipe_socket_dir`, `sieve_filter_socket_dir`, `sieve_execute_socket_dir` and same for other such settings. Otherwise not searchable.

Reply:
Expanded